src/rgw/rgw_op.cc

```diff
@@ -1287,6 +1287,7 @@ void RGWDeleteBucketTags::execute(optional_yield y)
   op_ret = retry_raced_bucket_write(this, s->bucket.get(), [this, y] {
     rgw::sal::Attrs attrs = s->bucket->get_attrs();
     attrs.erase(RGW_ATTR_TAGS);
+    s->bucket->set_attrs(attrs);
```

Comment:
if we're just setting the attrs so we can write them back, let's call `bucket->put_info()` directly instead of `merge_and_store_attrs()`

p.s. `get_attrs()` returns a mutable reference, so we don't need to copy the map or call `set_attrs()`:
```diff
-    rgw::sal::Attrs attrs = s->bucket->get_attrs();
+    rgw::sal::Attrs& attrs = s->bucket->get_attrs();
     attrs.erase(RGW_ATTR_TAGS);
-    s->bucket->set_attrs(attrs);
```

Reply:
Thanks Casey